flash/core/data/utilities/paths.py

```diff
@@ -133,6 +133,18 @@ def list_valid_files(
     return [path for path in paths if has_file_allowed_extension(path, valid_extensions)]
 
 
+def _show_filter_valid_files_message(
```

Comment:
I understand that this function would have existed as you started with the implementation using `rich`. But now, for simplicity, let's just copy this code to `filter_valid_files`. Nothing major, if you feel it's okay here, then all is good from me! :)

(maybe just rename it to something else if we agree that it should raise a warning and not print the message)

Reply:
I think this naming convention is okay in that is includes the triggering method's name, and what this particular method is responsible for handling.